internal/task/task/common/create_container.go

```diff
@@ -139,6 +139,12 @@ func getArguments(dc *topology.DeployConfig) string {
 		"raft_max_install_snapshot_tasks_num":  1,
 		"raft_use_fsync_rather_than_fdatasync": false,
 	}
+	if dc.ExistChunkFilePoolAllocatedPercent() {
+		chunkserverArguments["chunkFilePoolAllocatedPercent"] = dc.GetChunkFilePoolAllocatedPercent()
+	}
+	if dc.ExistChunkFormatThreadNum() {
+		chunkserverArguments["chunkFormatThreadNum"] = dc.GetChunkFormatThreadNum()
+	}
```

Comment:
Delete the function that determines whether it exists and use nil value for judgment.

May be we can use it:
if dc.GetChunkFilePoolAllocatedPercent() != 0 {
}

if dc.GetChunkFormatThreadNum() ！= 0 {
}

please push to the developer-activity branch.

Reply:
I think we can't judge whether it's empty by whether the return is 0 or not.